pom.xml

```diff
@@ -3,18 +3,20 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.15</version>
+        <version>4.18</version>
     </parent>
 
     <artifactId>sidebar-link</artifactId>
     <packaging>hpi</packaging>
     <name>Sidebar Link</name>
-    <version>1.12.1-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Sidebar-Link+Plugin</url>
+    <version>2.00.1</version>
```

Comment:
is it compatible with v1.12 ?

Reply:
I'm afraid not. 
I don't believe that I managed it to be compatible.